tests/utils/dockercli/volumelifecycle.go

```diff
@@ -171,6 +183,17 @@ func StopContainer(ip, containerName string) (string, error) {
 	return ssh.InvokeCommand(ip, dockercli.StopContainer+containerName)
 }
 
+// AttachVolume - attach volume to container on given host
```

Comment:
I guess this code block refers https://github.com/vmware/docker-volume-vsphere/pull/1286#discussion_r120119292

can you please refresh the method comment as per underlying util's code?